### ISSUE TITLE
Replace temporary Heading 3 icon in markdown visual edit mode

### ISSUE THREAD

**Issue by kenanyusuf** [FormidableLabs/spectacle-visual-editor#160]
We're using the wrong heading 3 icon in visual edit mode currently due to Evergeen UI not being up-to-date with the latest Blueprint icons.

<img width="695" alt="Screenshot 2021-11-22 at 09 53 11" src="https://user-images.githubusercontent.com/9557798/142840492-7e995b3c-4dce-476c-8855-ebd181dfbc7f.png">

I upgraded the icons in the Evergreen library, it should be included in the next release https://github.com/segmentio/evergreen/pull/1366.

We also need to upgrade Evergreen to v6 before we can make use of the new icon https://github.com/FormidableLabs/spectacle-visual-editor/issues/157.

Putting this on hold until both of the above are complete.
